Add compatibility with IE (Still needed for IE11)

meikoudras opened the pull request:
http://stackoverflow.com/questions/16618785/ie8-alternative-to-window-scrolly

vkbansal commented:
Sorry for the late response, I'm out on vacation. Will be back in couple of days and have a look at this.

Thanks for your contribution:)

meikoudras commented:
Thanks for the response. I added a few more changes:

I changed that the contextmenu event listener to be on the document, since I noticed some times when redrawing the dom the original dom element listener gets lost and the contextmenu listener is not fired.

I also noticed with multiple context menus touch screens have an issue clicking menu items ( the menu disappears and no action is fired). You can reproduce it here: http://vkbansal.github.io/react-contextmenu/#/multiple-menus by turning on chrome device mode and choosing a touch screen device (eg. Nexus 10). Hold down the left mouse button for the context menu to appear and try clicking the menu item.

vkbansal commented:
@meikoudras Can you make these as separate PRs ?
